scripts/es-integration-test.sh

```diff
@@ -96,14 +96,32 @@ teardown_storage() {
   docker compose -f "${compose_file}" down
 }
 
+build_local_img(){
+    local LOCAL_FLAG=''
+    local platforms="linux/amd64"
+    # Loop through each platform (separated by commas)
+    for platform in $(echo "$platforms" | tr ',' ' '); do
+      # Extract the architecture from the platform string
+      arch=${platform##*/}  # Remove everything before the last slash
+      make "build-binaries-$arch"
+    done
+    make create-baseimg
+    #bash scripts/build-upload-a-docker-image.sh ${LOCAL_FLAG} -b -c jaeger-es-index-cleaner -d cmd/es-index-cleaner -p "${platforms}" -t release -l Y
```

Comment:
Why can't we use this?

Reply:
well i tried it gives me a log message of Unable to find image
 `'localhost:5000/jaegertracing/jaeger-es-index-cleaner:local-test' locally`
although it is able to pull the image locally and run fine but i wanted to ask you first about it